src/OpenLoco/src/Objects/AirportObject.cpp

```diff
@@ -38,7 +38,7 @@ namespace OpenLoco
     }
 
     // 0x00490CAF
-    void AirportObject::load(const LoadedObjectHandle& handle, stdx::span<const std::byte> data)
+    void AirportObject::load(const LoadedObjectHandle& handle, stdx::span<const std::byte> data, ObjectManager::DependentObjects*)
```

Comment:
Not wild about adding the `ObjectManager::DependentObjects` parameter... But I suppose there is no avoiding it at this stage. Could put all three parameters into a session struct of sorts, perhaps, but I'm not sure that would be much nicer. Any thoughts?

Reply:
I thought of an alternative idea which is we get the dependent objects as a completely separate function. But can't do that whilst there are hooks in play.

Reply:
I thought of an alternative idea which is we get the dependent objects as a completely separate function. But can't do that whilst there are hooks in play.